cloud-init/postfix-setup.tpl.yml

```
@@ -1,7 +1,7 @@
 ---
 runcmd:
-  # Modify postfix's PRIMARY_DOMAIN in pca-gophish-composition docker-compose
-  # file to the email-sending domain for this assessment
+  # Modify postfix's PRIMARY_DOMAIN in pca-gophish-composition Docker
```

I think this improves the readability.

```suggestion
  # Modify postfix's PRIMARY_DOMAIN in the pca-gophish-composition Docker
```